tests/sentry/integrations/vercel/test_integration.py

```diff
@@ -118,3 +122,212 @@ def test_use_existing_installation(self):
         )
         self.assert_setup_flow(is_team=False)
         assert SentryAppInstallation.objects.count() == 1
+
+    @responses.activate
+    def test_update_organization_config(self):
+        """Test that Vercel environment variables are created"""
+        with self.tasks():
+            self.assert_setup_flow()
+        # mock org secret
+        responses.add(
+            responses.GET,
+            "https://api.vercel.com/v3/now/secrets/%s" % "sentry_org",
+            body=ApiError('The secret "%s" was not found.' % "sentry_org", code=404),
```

Comment:
@ceorourke I think you can potentially simplify the test by specifying the status to `404` and don't set the body to an ApiError. Then you can do something like:
```
secrets = ["sentry_org", "sentry_project_%s"%self.project.id, "next_public_sentry_dsn_%s"%self.project.id]
for secret in secrets:
             responses.add( responses.GET, "https://api.vercel.com/v3/now/secrets/%s" % secret , status=404)
```

Also, you should never rely on the ID of an object is a specific value. In this case, the project id being 1. That should always  be dynamically determined.